pages/instructorQuestionManualGrading/instructorQuestionManualGradingAbortInstanceQuestion.js

```diff
@@ -0,0 +1,25 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+const error = require('@prairielearn/prairielib/error');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', function(req, res, next) {
```

Comment:
How do we ever get to this code? I can't see that it's possible for a user to arrive here.

Reply:
This code has been eliminated. It's now part of the 'InstructorQuestionManualGrading.js' logic under the action 'abort_manual_grading'. It is now integrated property so this logic will be accessible.